utils/modCommon.js

```diff
@@ -137,7 +138,7 @@ const modCommon = {
     let { matches } = flagInfo;
     const { pingMods, snitch, flagReason, furtherInfo } = flagInfo;
 
-    const client = msg.client ?? member?.client;
+    const client = msg?.client ?? member?.client;
```

Comment:
client is possibly undefined.
```suggestion
    const client = msg?.client ?? member?.client;
    if (!client) return u.errorHandler("No client on flag for" + member)
```